spec/requests/form_templates_controller_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+RSpec.describe FormTemplatesController do
+  fab!(:user) { Fabricate(:user) }
+
+  before { SiteSetting.experimental_form_templates = true }
+
+  describe "#index" do
+    fab!(:form_template) { Fabricate(:form_template) }
+    fab!(:form_template_2) { Fabricate(:form_template, id: 1) }
```

Comment:
This may be flaky, not sure. May be worth specifying ids for all three fabricated templates.

Reply:
Or the opposite maybe? never specify the ID

Generally speaking specifying IDs is full of traps...

Reply:
Ah just merged it without seeing this comment. Sorry! Will address in follow up PR